kofta/public/locales/nl/translation.json

```
@@ -0,0 +1,222 @@
+			"listening": "aan het luisteren",
+			"toggleMuteKeybind": "Microfoon dempen schakelen toetsbinding",
+			"togglePushToTalkKeybind": "druk-om-te-praten toetsbinding"
+		},
+		"userVolumeSlider": {
+			"noAudioMessage": "geen mogelijkheid om geluid te ontvangen door een onbekende reden"
+		},
+		"addToCalendar": {
+			"add": "Toevoegen aan kalender"
+		},
+		"wsKilled": {
+			"description": "Webcontact is gestopt door de server. Dit gebeurd meestal als je een andere website bezoekt in een ander venster.",
+			"reconnect": "opnieuw verbinding maken"
+		},
+		"modals": {
+			"createRoomModal": {
+				"public": "publiek",
+				"private": "privé",
+				"roomName": "kamer naam",
+				"roomDescription": "kamer beschrijving",
+				"descriptionError": "maximale grote 500",
+				"nameError": "moet tussen de 2 en 60 karakters lang zijn"
+			},
+			"invitedToJoinRoomModal": {
+				"newRoomCreated": "Nieuwe Kamer Gemaakt",
+				"roomInviteFrom": "Kamer uitnodiging van",
+				"justStarted": "Zij zijn net begonnen",
+				"likeToJoin": ", wil je deelnemen?",
+				"inviteReceived": "je bent uitgenodigd voor"
+			},
+			"editProfileModal": {
+				"usernameTaken": "gebruikers naam niet beschikbaar",
+				"avatarUrlError": "Ongeldige foto",
+				"avatarUrlLabel": "Github/Twitter avatar link",
+				"displayNameError": "lengte 2 tot 50 karakters",
+				"displayNameLabel": "Weergavenaam",
+				"usernameError": "lengte 4 tot 15 tekens en alleen alfanumeriek / onderstrepingsteken",
+				"usernameLabel": "Gebruikersnaam",
+				"bioError": "maximale lengte van 160 tekens",
+				"bioLabel": "Bio"
+			},
+			"profileModal": {
+				"blockUserConfirm": "Weet je zeker dat je wilt voorkomen dat deze gebruiker lid wordt van een kamer die je ooit hebt gemaakt?",
+				"blockUser": "block user",
+				"makeMod": "make moderator",
+				"unmod": "moderator opheffen",
+				"addAsSpeaker": "toevoegen als luidspreker",
+				"moveToListener": "verplaatsen naar luisteraars",
+				"banFromChat": "verbannen van chatten",
+				"banFromRoom": "verbannen van kamer",
+				"goBackToListener": "terug naar luisteraars",
+				"deleteMessage": "dit bericht verwijderen"
+			},
+			"roomSettingsModal": {
+				"requirePermission": "toestemming nodig hebben om te spreken",
+				"makePublic": "ruimte openbaar maken",
+				"makePrivate": "ruimte privé maken"
+			}
+		}
+	},
+	"modules": {
+		"scheduledRooms": {
+			"title": "Geplande kamers",
+			"noneFound": "geen gevonden",
+			"allRooms": "alle geplande kamers",
+			"myRooms": "mijn geplande kamers",
+			"scheduleRoomHeader": "Plan kamer",
+			"startRoom": "start kamer",
+			"modal": {
+				"needsFuture": "moet in de toekomst zijn",
+				"roomName": "Kamer naam",
+				"minLength": "min lengte 2"
+			}
+		},
+		"roomChat": {
+			"title": "Chat",
+			"emotesSoon": "[emoticons binnenkort]",
+			"bannedAlert": "Je bent verbannen uit de chat",
+			"waitAlert": "U moet een seconde wachten voordat u weer een bericht verzendt",
```

```suggestion
			"waitAlert": "Je moet een seconde wachten voordat u weer een bericht verzendt",
```
Tot nu toe wordt overal je gebruikt. Dan is het raar om hier ineens u te gebruiken.